src/main/resources/institutional-config.properties

```diff
@@ -0,0 +1,144 @@
+app.context.name=kfs
+context.names.rice=cynergy
+production.environment.code=kfs-prod
+logs.directory=/infra/logs
+attachments.directory=/infra/work/attachments
+attachments.pending.directory=/infra/work/attachments/pending
+work.directory=/infra/work
+reports.directory=/infra/work/reports
+staging.directory=/infra/work/staging
+temp.directory=/infra/work/temp
+batch.file.lookup.root.directories=/infra/work/staging;/infra/work/reports
+templates.directory=/infra/work/templates
+attachment.dir.location=/infra/work/attachments
+staging.directory.sys.batchContainer=/infra/work/staging/sys/batchContainer
+data.xml.root.location=/infra/work/staging/workflow
+data.xml.pending.location=/infra/work/staging/workflow/pending
```

Comment:
There seems to be several properties that would benefit from "staging.directory" and/or "data.xml.root.location" placeholders. Would it make sense to use such placeholders for some of these directory-related properties, or are these all being defined without placeholders for a specific reason?